ui.frontend/src/GuideBridge.js

```diff
@@ -247,6 +249,10 @@ export default class GuideBridge {
         //TODO: implement it later. NO-OP for now.
     }
 
+    registerConfig(key, config) {
+        this.#userConfig[key] = config;
+    }
+
```

Comment:
where are we consuming this config?

Reply:
Hi @nit23uec ,
As per our last discussion with you and rishi, we decided to just store the configs for now .

Reply:
Config would be per form, GuideBridge instance is for page and a page can contain multiple forms.

Reply:
Config registered per formPath now

Reply:
we should use the strategy we use for other guidebridge APIs. (where formPath is taken from connect initialisation)
formpath should not be passed as arg in any of the other APIs.

Reply:
Also, till we are not consuming the configs I'd suggest not to expose this API, rather not call this API if form is v2.

Reply:
removed the api and added check to skip this api call from form app on the basis of formVersion